koku/koku/test_middleware.py

```diff
@@ -34,10 +35,25 @@
 from koku.middleware import HttpResponseUnauthorizedRequest
 from koku.middleware import IdentityHeaderMiddleware
 from koku.middleware import KokuTenantMiddleware
+from koku.middleware import KokuTenantSchemaExistsMiddleware
 from koku.middleware import RequestTimingMiddleware
 from koku.test_rbac import mocked_requests_get_500_text
 
 
+class KokuTenantSchemaExistsMiddlewareTest(IamTestCase):
+    def test_process_exception_empty_response(self):
+        """Assert that an exception raised for a non-existent Tenant returns empty result set."""
+        mock_request = MagicMock()
+        result = KokuTenantSchemaExistsMiddleware(Mock()).process_exception(mock_request, Exception)
```

Comment:
An exception instance needs to be passed in, not the class itself.
```suggestion
        result = KokuTenantSchemaExistsMiddleware(Mock()).process_exception(mock_request, Exception())
```

Reply:
It doesn't affect the test since the code under test is no longe doing an `isintance` check, but it is the correct thing to do.